packages/core/content-type-builder/package.json

```diff
@@ -77,7 +77,6 @@
   },
   "devDependencies": {
     "@strapi/pack-up": "4.15.5",
-    "@strapi/strapi": "4.15.5",
```

Comment:
Don't we need it anymore? What has changed?

Reply:
it'as actually never imported and the types are coming from another package